ruby/app.rb

```diff
@@ -0,0 +1,29 @@
+require 'sinatra'
+require 'jwt'
+
+METABASE_EMBEDDING_SECRET = ENV.fetch('METABASE_EMBEDDING_SECRET')
+METABASE_SITE_URL = ENV.fetch('METABASE_SITE_URL') || 'http://localhost:3000'
+METABASE_EMBED_DASHBOARD_ID = ENV.fetch('METABASE_EMBED_DASHBOARD_ID').to_i || 1
```

Comment:
You can use the second argument from `ENV.fetch` to provide a default value.

```suggestion
METABASE_EMBEDDING_SECRET = ENV.fetch('METABASE_EMBEDDING_SECRET') 
METABASE_SITE_URL = ENV.fetch('METABASE_SITE_URL', 'http://localhost:3000')
METABASE_EMBED_DASHBOARD_ID = ENV.fetch('METABASE_EMBED_DASHBOARD_ID', '1').to_i
```

Reply:
I didn't know this! TIL. Thank you!